main.py

```diff
@@ -332,13 +333,19 @@ def gen_code(kontaktdaten):
 
 
 def subcommand_search(args):
+    telegram = None
+    if args.t_token is not None and args.t_id is not None:
+        telegram = dict()
+        telegram["token"] = args.t_token
+        telegram["id"] = args.t_id
```

Comment:
Erscheint mir unnötig imperativ, da es ja auch funktional geht:

```suggestion
    if args.telegram_token is not None and args.telegram_id is not None:
        telegram = {
          "token": args.telegram_token,
          "id": args.telegram_id,
        }
```